src/Components/ItemProperties.jsx

```diff
@@ -34,7 +34,7 @@ export default function ItemProperties() {
     <div className={classes.propsContainer}>
       {
         Object.keys(element).length === 0 ?
-          <h2 className={classes.noElement}>No element selected</h2> :
+          <div className={classes.noElement}>No element selected</div> :
```

Comment:
Seems preferable to be moving towards semantic markup.  Ideally standard semantic sizes across the application so we don't have to normalize by hand everywhere.  I'm thinking something like this: https://material.io/blog/design-material-theme-type.  This will take time to get to, but if e.g. we have divs, we won't be able to search and replace for headers when we want to do this.